code-of-conduct.md

```diff
@@ -0,0 +1,76 @@
+
+To make clear what is expected, participants in list-related activities activities are expected to conform to the Code of Conduct. This Code of Conduct applies to all spaces related to the Expert Volunteer List, including, but not limited to, the private interactions between expert volunteers and scientists seeking support, the list entry submission form, the project's GitHub repository, list-related email, and list-related discussion in other public forums.
+
+The list's maintainers uphold the Code of Conduct and can be contacted by [email](expertvolunteerlist@gmail.com). All reports will be treated with confidentiality. Any report of a volunteer expert violating the code will lead to their removal from the list. The list's maintainers are not able to offer and explicitly dislaim all other forms of mediation or arbitration.
+
+##### Part 2. Expert Volunteer List - Code of Conduct
+
+The Research Support Network is dedicated to providing a welcoming and supportive environment for all people, regardless of background or identity. As such, we do not tolerate behaviour that is disrespectful to anyone involved in list-related activities or that excludes, intimidates, or causes discomfort to others. We do not tolerate discrimination or harassment based on characteristics that include, but are not limited to, gender identity and expression, sexual orientation, disability, physical appearance, body size, citizenship, nationality, ethnic or social origin, pregnancy, familial status, veteran status, genetic information, religion or belief (or lack thereof), membership of a national minority, property, age, education, socio-economic status, technical choices, and experience level. 
+
+Everyone who participates in list-related activities is required to conform to this Code of Conduct. It applies to all spaces related to the Expert Volunteer List, including, but not limited to, the private interactions between expert volunteers and scientists seeking support, the list entry submission form, the project's GitHub repository, list-related email, and list-related discussion in other public forums. By participating, participants indicate their acceptance of the procedures by which the list's maintainers resolve any Code of Conduct incidents, which may include storage and processing of their list-related information. 
+
+##### Part 2.1. Expected behaviour
+
+All participants in list-related activities are expected to show respect and courtesy to others. All interactions should be professional regardless of platform. In order to foster a positive and professional  environment we encourage the following kinds of behaviours in interactions:
+
+* Use welcoming and inclusive language
+* Be respectful of different viewpoints and experiences
+* Gracefully accept constructive criticism
+* Show courtesy and respect towards others
+
```

Comment:
I think including this from the landing page would be great: "Experts should be mindful of the fact that scientists seeking support may be living through (or have recently lived through) extremely traumatic experiences. Be considerate and avoid bringing up topics that may cause anxiety. Conversely, be prepared for the possibility of hearing accounts that could be disturbing. We recommend focusing on the science and if necessary setting clear boundaries on other topics."

Reply:
Thanks for your comments, @ElisabethKugler!

I've incorporated them in 39ab791 and also added you as an initial contributor in the "About" section.